src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -184,25 +184,14 @@ void AutomatedRssDownloader::loadRulesList()
 
 void AutomatedRssDownloader::loadFeedList()
 {
-    disconnectRuleFeedSlots();
-
-    const Preferences *const pref = Preferences::instance();
-    const QStringList feed_aliases = pref->getRssFeedsAliases();
-    const QStringList feed_urls = pref->getRssFeedsUrls();
-    ui->listFeeds->clear();
-
-    for (int i = 0; i < feed_aliases.size(); ++i) {
-        QString feed_url = feed_urls.at(i);
-        feed_url = feed_url.split("\\").last();
-        qDebug() << Q_FUNC_INFO << feed_url;
-
-        QListWidgetItem *item = new QListWidgetItem(feed_aliases.at(i), ui->listFeeds);
-        item->setData(Qt::UserRole, feed_url);
-        item->setFlags(item->flags() | Qt::ItemIsUserCheckable | Qt::ItemIsTristate);
+    if (Rss::ManagerPtr manager = m_manager.toStrongRef()) {
```

Comment:
>Point of mine is that the if-block creates a whole pile of indentations (also the block is spread all across all this method scope) is meaningless, using the if-false-branch-out is much more elegant IMO, you figure out what happens in false condition much faster without reading to the bottom of the if-block.

I agree with that.
I thought you argue specifically against statement in parentheses (assignment alongside with checking). GitHub does not show to me what comes after annotated lines.
But I fully support the "early returns" in such cases.